sklearn/linear_model/tests/test_logistic.py

```diff
@@ -1307,3 +1307,18 @@ def test_warm_start_converge_LR():
         lr_ws.fit(X, y)
     lr_ws_loss = log_loss(y, lr_ws.predict_proba(X))
     assert_allclose(lr_no_ws_loss, lr_ws_loss, rtol=1e-5)
+
+
+def test_log_reg_scoring_path_multinomial():
+    # Make sure that the scores are all different for different C values
```

Comment:
At the moment you are only checking that some are different. And if so, you only need to test two values of C